widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidget.kt

```diff
@@ -0,0 +1,31 @@
+package com.glia.widgets.entrywidget
+
+import android.app.Activity
+import android.content.Context
+import android.view.View
+import com.glia.widgets.launcher.ActivityLauncher
+
+/**
+ * An interface for interacting with the entry widget(pre-built UI for engagement type selection).
+ */
+interface EntryWidget {
+    /**
+     * Shows the entry widget.
+     *
+     * @param activity The Activity used to show the entry widget
+     */
+    fun show(activity: Activity)
+
+    /**
+     * Creates the entry widget view that might be easily embedded in a custom UI.
+     *
+     * @return The entry widget view.
+     */
+    fun embeddableView(context: Context): View
```

Comment:
According to the project plan, the name should be `createEntryWidgetView()`, shouldn't it?

Reply:
Is it okay that there is no `EntryWidget().hide()` interface at the moment?

Reply:
> According to the project plan, the name should be createEntryWidgetView(), shouldn't it?
I will change it as follows.

> Is it okay that there is no EntryWidget().hide() interface at the moment?
I'm not sure if we need it at all. In which scenario will we need it?

Reply:
Hm, good question. For example, some an integrator might need to hide the Entry Widget when some event happens. Not sure, though. Honestly.

Might make sense for iOS. @rasmustautsglia Can you think of the cases when iOS SDK integrator might need tht E`ntryWidget.hide()` interface?

Reply:
The only case I see when the visitor might want to close the `EntryWidget,` in this case, every Android user knows how to close the `BottomSheet.`